### PR TITLE
Stabilize spec 3.1.0

### PR DESCRIPTION
**Pull request by bgilbert**

### CODE REVIEW

File: doc/development.md [hunk 1]
```diff
@@ -118,39 +118,49 @@ When an experimental version of the Ignition config spec (e.g.: `3.1.0-experimen
...
+- Update all `config/vX_Y` imports in `config/vX_(Y+1)_experimental` to `config/vX_(Y+1)_experimental`
 - Update `MaxVersion` in `config/vX_(Y+1)_experimental/types/config.go` to have the correct major/minor versions and `PreRelease` set to `"experimental"`
-- Update `config/vX_(Y+1)_experimental/config.go` to use `config/vX_Y` for parsing
-- Update `config/vX_(Y+1)_experimental/config_test.go` to test that the new stable version is valid, the new experimental version is valid, and the old experimental version is invalid
-- Update `config/vX_(Y+1)_experimental/translate/translate.go` to only set the new version.
-- Update `config/config.go` to handle the new config.
+- Update `config/vX_(Y+1)_experimental/config_test.go` to test that the new stable version is invalid and the new experimental version is valid
+- Update `config/vX_(Y+1)_experimental/translate/translate.go` to translate from the previous stable version.  Update the `old_types` import, delete all functions except `translateIgnition` and `Translate`, and ensure `translateIgnition` translates the entire `Ignition` struct.
+- Update `config/config.go` to handle the new stable and experimental versions.
```
Comment (arithx) [inline doc/development.md:138]: Should this be split into two lines, one in `Making the experimental package stable` for adding the new stable version and then leave only the new experimental version portion here?

File: config/v3_2_experimental/config.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Red Hat, Inc.
```
Comment (arithx) [inline config/v3_2_experimental/config.go:1]: Most of these files have different / non-current years.

File: doc/development.md (hunk 1 above)
Comment (bgilbert) [reply doc/development.md:138]: Could do, but I don't think it's worth it.  The `config.go` changes are somewhat intertwined, and the code isn't expected to compile at this point anyway.

File: config/v3_2_experimental/config.go (hunk 2 above)
Comment (bgilbert) [reply config/v3_2_experimental/config.go]: Dates updated.

**Comment by bgilbert** [comment]
retest this please

**Pull request merged**